web/routes/admin/transactions-overview.js

```diff
@@ -0,0 +1,27 @@
+var express = require('express')
+var router = express.Router()
+const url = require('../../settings')
+const axios = require('axios')
+const middleware = require('../middleware/middleware.js')
+const role = 'ADMIN'
+
+router.get('/overzicht/transacties', middleware.authRole(role), function (req, res) {
+    const get_transactions_url = `${url}/users/admin/transactions`
+    const token = req.signedCookies.Authorization
+
+    axios
+        .get(get_transactions_url, {
+            headers: {
+                Authorization: `${token}`,
+            },
+        })
+        .then((response) => {
+            if (response.data) {
```

Comment:
```suggestion
            if (response.data) {
```
response.data.transactions